dist/BrazeKit.iife.js

```diff
@@ -9172,7 +9172,7 @@ var mpBrazeKitV4 = (function (exports) {
 	var name = 'Appboy',
 	    suffix = 'v4',
 	    moduleId = 28,
-	    version = '4.1.0',
+	    version = '4.1.1',
```

Comment:
Please undo all changes in this file as well.

Reply:
This change is also already [in the master-v4 branch](https://github.com/mparticle-integrations/mparticle-javascript-integration-braze/blob/b043d4637a7e0b6d82bf550d63936a77d6b2e0c2/dist/BrazeKit.iife.js#L9175).